wallets/provider-walletconnect-2/src/session.ts

```diff
@@ -87,7 +86,10 @@ export async function createSession(
 
       onCloseModal = new Promise((_, reject) => {
         modal.subscribeModal((state) => {
-          // the modal was closed so reject the promise
+          /*
+           * the modal was closed so reject the promise
+           * TODO CHECK REJECT UNHANDLED ERROR
```

Comment:
What do you mean by this? Avoiding sending modal rejection to sentry?

Reply:
I will remove the TODO. I think we have a bug in embedded in handling connect errors. (ignore first 25 seconds of the video)


https://github.com/rango-exchange/rango-client/assets/99180934/6245663f-b05f-42a8-94c7-511c18d1603f

Reply:
So please make sure we created an issue on our backlog.